app/redux/reducers/patientListFilters.js

```diff
@@ -0,0 +1,22 @@
+import update from 'immutability-helper';
+import * as actionTypes from '../constants/actionTypes';
+
+const initialState = {
+  isPatientListVisible: false,
+  patientListSearchTextInput: '',
+}
```

Comment:
I'm a little torn on this one. On one hand I do see the value/desire to keep the initial state colocated with the rest of the code, but on the other I do like the self-documenting nature of the `initialState.js` that effectively shows what the whole Redux state tree should look like in the application. I don't have _strong_ feelings about it, and in the RTK world, this pattern feels more idiomatic and since that's where we're 🤞🏼 headed, I'm inclined to let this be the start of the colocation.

Reply:
Ahh! This was not an intentional decision I made, just my inattention, and we used RTK at my previous role so I probably just put it here by muscle memory. Funny, I think we talked about this during my interview with you. If you're good with this, then perhaps we can leave it here and start the co-location.